File: core-java-practice/gcr-codebase/PowerCalculation.java
import java.util.Scanner;

public class PowerCalculation {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // Input base and exponent
        System.out.print("Enter base: ");
        double base = sc.nextDouble();

        System.out.print("Enter exponent: ");
        double exponent = sc.nextDouble();

        // Calculate power
        double result = Math.pow(base, exponent);

        // Output result
        System.out.println("Result = " + result);

      
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/PowerCalculation.cs
class PowerCalculation
{
    static void Main(string[] args)
    {
        // Input base and exponent
        Console.Write("Enter base: ");
        double baseValue = Convert.ToDouble(Console.ReadLine());

        Console.Write("Enter exponent: ");
        double exponent = Convert.ToDouble(Console.ReadLine());

        // Calculate power
        double result = Math.Pow(baseValue, exponent);

        // Output result
        Console.WriteLine("Result = " + result);
    }
}
